### PR TITLE
chore: bump up etcd chart version from 6.2.3 to 8.6.0

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Signed-off-by: Abhinandan Purkait <abhinandan.purkait@mayadata.io>

### CODE REVIEW

**Review by avishnu — APPROVED** [review]
lgtm

File: chart/values.yaml [hunk 1]
```diff
@@ -325,6 +325,7 @@ etcd:
 
   auth:
     rbac:
+      create: false
```
Comment (avishnu) [inline chart/values.yaml:328]: What is this for?

Comment (Abhinandan-Purkait) [reply chart/values.yaml:328]: This was needed to disable auth otherwise etcd-client doesn't work in core-agent.

Comment (avishnu) [reply chart/values.yaml:328]: Ok

**Review by tiagolobocastro — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**